test/integration/logic/resolvable/UnifyAttributeConcludableTest.java

```diff
@@ -129,16 +130,15 @@ private Rule createRule(String label, String whenConjunctionPattern, String then
         }
     }
 
-    @Ignore // TODO enable when we have the final structure of conclusions and concludable in place
     @Test
     public void literal_predicates_unify_and_filter_answers() {
-        String conjunction = "{ $a > 'b'; $a < 'y'; }";
-        Set<Concludable<?>> concludables = Concludable.create(parseConjunction(conjunction));
+        String conjunction = "{ $a = 'john'; }";
```

Comment:
Scrutinise this! Changed given our conclusion on how we should handle ValueConstraints other than `EQ`